a10_octavia/controller/worker/tasks/a10_network_tasks.py

```diff
@@ -825,14 +825,14 @@ def revert(
         if vrid_list:
             vrid_floating_ip_list = [vrid.vrid_floating_ip for vrid in vrid_list]
 
-        if vrid_floating_ip_list:
-            vrid_value = CONF.a10_global.vrid
-            try:
-                self.update_device_vrid_fip(
-                    vthunder, vrid_floating_ip_list, vrid_value)
-            except Exception as e:
-                LOG.error("Failed to update VRID floating IPs %s due to %s",
-                          vrid_floating_ip_list, str(e))
+            if vrid_floating_ip_list:
```

Comment:
I think this will not be needed, since HandleVRIDFloatingIP expects vrid_list as iterable object. We need to add the change in GetVRIDForLoadbalancerResource to return empty vrid_list. We don't need to add null check. I will be adding changes for this through      https://a10networks.atlassian.net/browse/STACK-2244 bug fix

Reply:
removed